app/controllers/api/autocomplete_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
```

Comment:
@dweinberger is going to love this :)

@dweinberger -- this is the first class in the API for Green Commons -- we can start building out additional API endpoints from here on out, but this is our first!